lib/active_cached_resource/collection.rb

```diff
@@ -0,0 +1,28 @@
+module ActiveCachedResource
+  class Collection < ActiveResource::Collection
+    private
+
+    def request_resources!
+      return @elements if requested? || resource_class.nil?
```

Comment:
when is the resource_class nil?

Reply:
when initializing the model and not assigning `resource_class` . Its an attribute not in the constructor args.